controllers/toolchainconfig/configuration.go

```diff
@@ -64,6 +64,10 @@ func ForceLoadToolchainConfig(cl runtimeclient.Client) (ToolchainConfig, error)
 	return newToolchainConfig(config, secrets), nil
 }
 
+func NewToolchainConfig(config *toolchainv1alpha1.ToolchainConfigSpec, secrets map[string]map[string]string) ToolchainConfig {
```

Comment:
this function doesn't seem to be used, unless I'm missing something

Reply:
Yep, you are right. It's not used anymore. Removed in https://github.com/codeready-toolchain/host-operator/pull/1048/commits/d53e5d4dfb9c7b9be812223f846f241a3107c3cd